dd-bridge-android/src/main/kotlin/com/datadog/android/bridge/internal/BridgeSdk.kt

```diff
@@ -31,10 +31,11 @@ internal class BridgeSdk(
         val nativeConfiguration = buildConfiguration(configuration)
         val trackingConsent = buildTrackingConsent(configuration.trackingConsent)
 
-        datadog.setVerbosity(Log.VERBOSE)
         datadog.initialize(appContext, credentials, nativeConfiguration, trackingConsent)
 
         datadog.registerRumMonitor(RumMonitor.Builder().build())
+
+        configureSdkVerbosity(configuration)
```

Comment:
shouldn't we set verbosity before `datadog.initialize` call? some messages can be printed during `initialize` step.

Reply:
Indeed